purrr/users.py

```diff
@@ -26,6 +71,10 @@ def github_username_to_slack_user_id(github_username: str) -> str:
     Returns:
         Slack user ID, or "" if not found.
     """
+    # Don't even check GitHub teams, only individual users.
+    if "/" in github_username:
+        return ""
+
     # Optimization: if we already have it cached, no need to look it up.
     slack_user_id = data_helper.cached_slack_user_id(github_username)
```

Comment:
This is not very likely, but say a user changes their username?
Do we just not support that?

Reply:
I'm not sure a Slack user **can** change their username, only their profile details (e..g **display** name). Even if it is possible, I'm fine with the user having to opt-out again in this unusual situation, or we could create a "good first issue" to fix this.

Reply:
I meant a github username. I don't think you can change a Slack username.

Reply:
Yeah, you're right, but same answer: I'm fine with the user having to opt-out again, if they change their GitHub username. Not only is this rare, it's even less likely when the user is already a contributing member in a project/company.